### PR TITLE
Feat/edit

### PR DESCRIPTION
**Pull request by zara-sh**

### CODE REVIEW

**Comment by zara-sh** [comment]
@Rolune 
would you review my code before merging to main?

**Comment by Rolune** [comment]
Other than the stuff I commented about,
Does this update the tasks object? Still haven't got how React handles object update 100%, would need to test around to see.

Other than that, looks great!

**Comment by Rolune** [comment]
Oh, also can't help with site deployment. We'll figure that out as we go.

File: src/App.js [hunk 1]
```diff
@@ -29,53 +20,26 @@ function App() {
...
-      }
-      // for name, this is temp to test function
-      let newTasks = tasks;
-      newTasks.splice(tIndex, 1);
-      setTasks([...newTasks]);
-    }
+    const tIndex = tasks.findIndex(({ id }) => id === taskToRemove.id);
+    const newTasks = [...tasks];
+    newTasks.splice(tIndex, 1);
+    setTasks(newTasks);
```
[the remark this answers is not in the document]
Comment (Rolune) [reply src/App.js:27]: Just checked and it works

**Review by Rolune — APPROVED** [review]
Nothing that looks like it'll break this

**Pull request merged**